package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "tslint \"src/**/*.ts\"",
     "test": "ng test",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor"
+    "e2e": "protractor",
```

Comment:
these changes are not relevant for this issue.
can you push these changes to relevant issue ?